src/utilities/window.ts

```diff
@@ -0,0 +1,9 @@
+import { globalExists } from './global'
+
+export function getComputedStyleRecord(element: Element | undefined): CSSStyleDeclaration | undefined {
```

Comment:
Is there a particular reason for the Record suffix? We don't generally add the return type to our methods.

Reply:
Just left over from when i was changing the value into a record. I'll fix

Reply:
Although it should be different. Otherwise vscode will assume we want the method from `window`

Reply:
Does it really? That's dumb.

Reply:
it looks like when I import it, it does override the window. But the fact that it can appear to work when not imported because it's actually using the version from `window` makes me feel like a slightly different name is a better route

https://user-images.githubusercontent.com/6098901/190163096-b3f6a3b8-9906-41d3-bfc5-d19871ca6218.mov